arangograph/single-sign-on.md

```diff
@@ -9,9 +9,28 @@ title: Single Sign-On (SSO)
 This feature is only available in ArangoGraph Enterprise.
 {% endhint %}
 
-ArangoGraph provides support to enable Single Sign-On (SSO) authentication using
-SAML 2.0 with SCIM provisioning. This enables you to propagate to ArangoGraph any
-user access changes by using the dedicated API.
+ArangoGraph provides support to enable **Single Sign-On** (SSO) authentication
+using **Security Assertion Markup language 2.0** (SAML 2.0) with the
+**System for Cross-domain Identity Management** (SCIM) provisioning. 
+This enables you to propagate to ArangoGraph any user access changes by using
+the dedicated API.
```

Comment:
Can we change how this is positioning SCIM?

Having read through the docs it reads as though you have to use SCIM to setup SSO, whereas the reality is that we don't want people to use SCIM if we can avoid it, it's an optional extra that most do not need.
Perhaps we can have the SSO SAML page (this one) with a child page of SCIM, or just some other way to get the point across to users that "If you want to configure SSO, do these steps then stop here. If you then want SCIM on top, this is where you can go".

Reply:
Good to know that we want to avoid enabling SCIM. In this case, we can move the sections "About SCIM" and "How to enable SCIM in Okta" to a new page (child of SSO SAML).

Reply:
Agree, right now it all reads like it's required. @nerpaula let's move the SCIM topic out of the main content here.